packages/widget/README.md

```diff
@@ -0,0 +1,55 @@
+# Sygmaprotocol Lit Widget
+
+### Dependencies
+ 
+To integrate the Widget into any project that uses web components you will need to add the following dependencies:
+
+```bash
+yarn add @buildwithsygma/sygmaprotocol-widget @buildwithsygma/sygma-sdk-core
+```
+
+### Code integration
```

Comment:
isnt this strictly lit specific documentation

Reply:
would we like to add vanilla js example as well

Reply:
I don't see why we can have this here. Usually docs and readmes would share steps

Reply:
@saadjhk web component example?

Reply:
yes I think shouldnt it be a normal JS example? lit is the framework we are using but not necessarily would be the go to for users